calico/felix/splitter.py

```diff
@@ -33,18 +33,19 @@ class UpdateSplitter(Actor):
     and IPv6-specific actors.
     """
     def __init__(self, config, ipsets_mgrs, rules_managers, endpoint_managers,
-                 iptables_updaters):
+                 iptables_updaters, ipv4_nat_manager):
```

Comment:
This is a very unclear variable name - the caller passes 'v4_masq_manager' to this, but also has a 'v4_nat_updater'.  My first reading was that this would be the latter (which wouldn't work), not the former.  I suggest using [ip]v4_masq_manager or similar instead.